app/src/main/java/com/vero/tasky/core/data/local/PreferencesImpl.kt

```diff
@@ -0,0 +1,33 @@
+package com.vero.tasky.core.data.local
+
+import android.app.Application
+import android.content.Context
+import androidx.core.content.edit
+import com.vero.tasky.core.domain.local.Preferences
+import com.vero.tasky.core.domain.model.User
+import javax.inject.Inject
+
+class PreferencesImpl @Inject constructor(
+    application: Application
+) : Preferences {
+
+    private val pref = application.getSharedPreferences(
+        "com.vero.tasky", Context.MODE_PRIVATE)
```

Comment:
You are going to store a user's JWT token

Meaning, if someone get access to it, they can modify a user's information
You don't want that

For that, you can use an EncryptedSharedPreference